### DISCUSSION TITLE
Does RefindPlus Include Fixes for Serial Input?

### DISCUSSION THREAD

**Discussion by hradec** [dakanji/RefindPlus#61] [General]
**Is your feature request related to a problem? Please describe.**

Running Refind in textmode on a GCE (Google Compute Environment) instance, I can see the menu in the serial connection, but I can't select any options or interact with it any way.

**Describe the solution you would like**

I'm wondering if there is any fixes in this fork regarding serial input.

**Comment by dakanji** [comment]
> I'm wondering if there is any fixes in this fork regarding serial input.

There is nothing extra beyond whatever is in rEFInd specifically done here for such input.
